scenario/migrate.yaml

```diff
@@ -18,7 +18,7 @@ process:
           - act_get_info_images_src: True
           - act_get_info_inst_src: True
           - act_get_info_volumes_src: True
-          - act_get_info_objects_src: True
+          - act_get_info_objects_src: False
```

Comment:
`act_get_info_objects_src` and `act_get_info_objects_dst` seem to be unrelated to the patch